jobs/e2e-test.groovy

```diff
@@ -338,7 +359,11 @@ onWorker(WORKER_TYPE, '5h') {     // timeout
          // tests failed.  That is, this will succeed as long as there
          // are no lambdatest framework errors; it's up to use to look
          // for actual smoketest-code errors in analyzeResults(), below.
-         runLambdaTest();
+         parallel([
+               "run-lambdatest": { runLambdaTest(); },
+               "test-cycloud": { runCypressCloud(); },
+            ]);
+         // runLambdaTest();
       }
 
       stage("Analyzing results") {
```

Comment:
```
// Collect results from the triggered job
        stage("Collect Cypress Results") {
            def resultsDir = "results";
            def mergedResults = "${resultsDir}/junit-merged.xml";

            sh("mkdir -p ${resultsDir}");

            // Unstash results from all workers
            for (int i = 0; i < params.NUM_WORKER_MACHINES.toInteger(); i++) {
                unstash name: "worker-${i}-results";
            }

            // Merge results into a single JUnit XML file
            sh("""
                find ${resultsDir} -name '*.xml' -print0 | xargs -0 cat > ${mergedResults}
            """);

            echo("Merged results written to ${mergedResults}");
        }
```

Reply:
Is this how results get merged for unit tests too? If not, what's different and why?

Reply:
I think it’s a slightly different alternative way of how we might collect junit results - maybe more straightforward.